suite-native/assets/src/assetsSelectors.ts

```diff
@@ -73,25 +82,66 @@ export const selectDeviceAssetsWithBalances = (state: AssetsRootState & DeviceRo
         };
     });
 
-    return deviceNetworksWithAssets.map((networkSymbol: NetworkSymbol) => {
+    let totalFiatBalance = 0;
+
+    const assets = deviceNetworksWithAssets.map((networkSymbol: NetworkSymbol) => {
         const networkAccounts = accountsWithFiatBalance.filter(
             account => account.symbol === networkSymbol,
         );
         const assetBalance = networkAccounts.reduce(
-            (sum, { cryptoValue }) => sum.plus(cryptoValue),
-            new BigNumber(0),
+            (sum, { cryptoValue }) => sum + Number(cryptoValue),
```

Comment:
For fiat value it's ok to loose the precision, but not sure about crypto value 🤔

Reply:
If it's rounded to 8 decimals anyway, I guess it's fine.